index.html

```diff
@@ -12,7 +12,46 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
-
+    
+    <nav class="navbar">
+        <div class="container">
+            <img class="logo" src="img/karma-logo.svg" width="26">
+        <ul class="links">
+            <li>Meet karma</li>
+            <li>How it works</li>
+            <li>Store</li>
+            <li>Blog</li>
+            <li>Help</li>
+            <li>Login</li>
+        </ul>
+    
+    </nav>
+</div>
+    <div class="aside">
+        <img src="img\first-background.jpg" alt="abstract wifi cook" style="width:100%;">
+        <div class="centered">
+            <h1> Introducing Karma</h1>
+            <h2>Bring wifi with you, everywhere you go</h2>
+            <button >Learn More</button>
+        </div>
+        </div>
+        <h2>Everyone needs a little Karma.</h2>
+       <!--block of icons-->
+    <ul class="icontainer">
+    
+     <li> <img  src="img\icon-devices.svg" alt="Internet for all devices" width=127 height=127>
+     <h3> Internet for all devices</h3></li>
+     <li><img  src="img\icon-coffee.svg" alt= "Boost your productivity" width=127 height=127>
```

Comment:
What your HTML indentation here. It should be easy to see what is inside what, from the spacing / indentation of your code.